doc/source/user_guide/style.ipynb

```diff
@@ -854,10 +844,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "(df.style\n",
-    "   .set_na_rep(\"FAIL\")\n",
-    "   .format(None, na_rep=\"PASS\", subset=[\"D\"])\n",
-    "   .highlight_null(\"yellow\"))"
+    "(\n",
+    "    df.style.set_na_rep(\"FAIL\")\n",
+    "    .format(None, na_rep=\"PASS\", subset=[\"D\"])\n",
+    "    .highlight_null(\"yellow\")\n",
```

Comment:
We should maybe decide whether for those specific "method-chaining" cases we want to strictly follow black or not (it's quite typical for pandas, and the original layout might make it clearer what is going on)